tests/Fixtures/Integration/misc/PHP8_0.test

```diff
@@ -0,0 +1,371 @@
+--TEST--
+PHP 8.0 test.
+--RULESET--
+{
+    "@PHP80Migration": true,
+    "@PHP80Migration:risky": true,
+    "@PhpCsFixer": true,
+    "@PhpCsFixer:risky": true,
+    "phpdoc_to_return_type": true
+}
+--REQUIREMENTS--
+{"php": 80000}
+--EXPECT--
+<?php
+
+declare(strict_types=1);
+
+// https://wiki.php.net/rfc/named_params
+array_fill(start_index: 0, num: 100, value: 50);
+htmlspecialchars($string, double_encode: false);
+array_slice($array, $offset, $length, preserve_keys: true);
+new ParamNode($name, variadic: $isVariadic, byRef: $passByRef);
+array_foobar(array: $value); // reserved keyword as parameter name!
+
+// https://wiki.php.net/rfc/nullsafe_operator
+// TODO - CRASHING !!!
+// $country = $session?->user?->getAddress()?->country;
+
+// https://wiki.php.net/rfc/attributes_v2
+// https://wiki.php.net/rfc/attribute_amendments
+// https://wiki.php.net/rfc/shorter_attribute_syntax
+// https://wiki.php.net/rfc/shorter_attribute_syntax_change
+// TODO - BREAKING CODE !!!
+/*
+#[MyAttribute]
+#[\MyExample\MyAttribute]
+#[MyAttribute(1234)]
+#[MyAttribute(value: 1234)]
+#[MyAttribute(MyAttribute::VALUE)]
+#[MyAttribute(array("key" => "value"))]
+#[MyAttribute(100 + 200)]
+#[Attribute(Attribute::TARGET_METHOD | Attribute::TARGET_FUNCTION)]
+#[MyAttribute(1234), MyAttribute(5678)]
+class PostsController
+{
+    #[Route("/api/posts/{id}", methods: ["GET"])]
+    public function get($id): void
+    {
+    }
+}
+
+$object = new #[ExampleAttribute] class ()
+{
+    #[ExampleAttribute]
+    public function foo(#[ExampleAttribute] $bar): void
+    {
+    }
+};
+*/
+
+// https://wiki.php.net/rfc/union_types_v2
+class Number
+{
+    // TODO - CRASHING !!! - next line is wrongly modified to `private int | float | public null $number;`
```

Comment:
@SpacePossum , #5405 done the fix for method params, but not for class properties. can you take a look?

Reply:
done: https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/5439